src/OpenColorIO/Baker.cpp

```diff
@@ -388,6 +395,16 @@ OCIO_ADD_TEST(Baker_Unit_Tests, bake)
 
 }
 
+OCIO_ADD_TEST(Baker_Unit_Tests, bake_empty_config)
```

Comment:
[Minor] To better follow other unit test names, the 'major' name is the class name i.e. 'Baker', and the 'minor' name is the test itself i.e. 'empty_config' or 'missing_config'.

Reply:
Got it, thanks! Changed the major name to `Baker` for both the `bake` and `empty_config` tests.